src/include/fsal_api.h

```diff
@@ -412,7 +412,7 @@ typedef enum {
 } stats_ops;
 
 
-#define FSAL_STAT_DATA_SIZE 64
+#define FSAL_STAT_DATA_SIZE 72
```

Comment:
Why not a factor of block or power of 2?

Reply:
I just set it to the exact size of the struct that should fit in. It shouldn't create alignment problems (I think).

Reply:
Ah ok. That'd be useful to have a `static_assert` then ?

Reply:
Yes there is already a `static_assert` on the ring side that checks that `sizeof` this array is `>=` `sizeof` the `struct` that must fit in it.

Reply:
it's used and allocated within that struct: https://github.com/scality/nfs-ganesha/blob/V3-stable-scality/src/include/nfs_proto_data.h#L335